Sources/TuistCore/Graph/Graph.swift

```diff
@@ -185,6 +186,13 @@ public class Graph: Graphing {
     public var targets: [TargetNode] {
         return cache.targetNodes.flatMap { $0.value.values }
     }
+    
+    public func target(path: AbsolutePath, name: String) -> TargetNode? {
+        guard let targetNode = findTargetNode(path: path, name: name) else {
```

Comment:
nitpick: we can directly return `findTargetNode(path: path, name: name)`

Reply:
Thanks, will change this